minimum go version needed

Ran this on my Raspberry Pi running this:

```
> cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

and got these results

```
> go run .
go: downloading golang.org/x/crypto v0.1.0
# golang.org/x/crypto/ssh
/home/emv/go/pkg/mod/golang.org/x/crypto@v0.1.0/ssh/cipher.go:499:13: undefined: io.Discard
/home/emv/go/pkg/mod/golang.org/x/crypto@v0.1.0/ssh/session.go:508:14: undefined: io.Discard
/home/emv/go/pkg/mod/golang.org/x/crypto@v0.1.0/ssh/session.go:521:14: undefined: io.Discard
note: module requires Go 1.17
```

I'll install a suitably current Go next.

--- a/README.md
+++ b/README.md
@@ -12,4 +12,6 @@ Known affected versions:
 * Fedora 36 `OpenSSH_8.8p1, OpenSSL 3.0.5 5 Jul 2022`
 * macOS Ventura
 
-To test your version of `ssh`, run: `go run .`
+To test your version of `ssh`, run: `go run .` 
+
+You need Go 1.17 or newer to build this package.

